repositories/analytics_views/apikeys.sql

```diff
@@ -0,0 +1,10 @@
+DROP VIEW IF EXISTS analytics.apikeys;
+
+CREATE VIEW analytics.apikeys WITH (security_invoker = false) AS
+SELECT
+      id,
+      org_id AS organization_id,
```

Comment:
Only level of normalization I did here was to use organization_id instead of org_id everywhere, for clarity